bin/generate

```diff
@@ -0,0 +1,150 @@
+#!/bin/bash
+
+# A script for generating `namespaces.yaml` and the associated LookML.
+# This repository builds namespaces.yaml from Glean applications and
+# `custom-namespaces.yaml`, and then generates files and LookML that
+# match the specification in `namespaces.yaml` and table definitions
+# in BigQuery. The resulting LookML is pushed to our `looker-hub`
+# repository.
+#
+# Environment variables:
+#   HUB_SSH_KEY_BASE64: A base64-encoded ssh secret key with permissions to push
+#                       to looker-hub.
+#   HUB_REPO_URL:       The URL to the looker-hub repository.
+#                       Requires the SSH format, e.g. git@github.com:user/repo.git
+#   HUB_BRANCH_SOURCE:  The source branch for generating LookML.
+#                       Defaults to 'base'. Files present the source
+#                       branch will remain unchanged by generation.
+#   HUB_BRANCH_PUBLISH: The destination branch for publishing LookML.
+#                       Defaults to 'test-lookml-generation'. If the
+#                       branch doesn't exist, creates it from source.
+#   GCLOUD_SERVICE_KEY: The service key used to authenticate with GCP.
+#                       Needs to have read access to all tables specified
+#                       in `namespaces.yaml`. Not required in local dev.
+#
+# Example usage:
+#   export HUB_SSH_KEY_BASE64=$(cat ~/.ssh/id_rsa | base64)
+#   make build && make run
+
+HUB_REPO_URL=${HUB_REPO_URL:-"git@github.com:mozilla/looker-hub.git"}
+HUB_BRANCH_SOURCE=${HUB_BRANCH_SOURCE:-"base"}
```

Comment:
`main`?

Reply:
Looker is configured to deploy from `main`, so when we publish this the first time, the `main` branch will have the generated schemas + files from base, and base will have the things we haven't yet gotten around to adding to the generation pipeline.